compiler/msgs.nim

```diff
@@ -9,7 +9,7 @@
 
 import
   options, strutils, os, tables, ropes, platform, terminal, macros,
-  lineinfos
+  lineinfos, strformat
```

Comment:
Hm, since strutils is already imported, should the formatting procedures from that module be used?

Reply:
what's the downside of depending on strformat?
Happy to change code if there is any concrete downside; if not, fmt leads to simpler code

Reply:
We don't use ``fmt`` in the compiler because I enjoy quick turnaround times / faster compiles (which are actually quite bad already). Also after a colon a space follows, just like in written English.

Reply:
done